components/service/glean/metrics.yaml

```diff
@@ -66,6 +66,21 @@ glean.internal.metrics:
       - telemetry-client-dev@mozilla.com
     expires: never
 
+  android_sdk_version:
+    type: string
+    lifetime: application
+    send_in_pings:
+      - glean_client_info
+    description: |
+      The optional Android specific SDK version of the software running on this hardware device.
+    bugs:
+      - 1525606
+    data_reviews:
+      - TODO
```

Comment:
This will need to change once we receive data-review+ (see [here](https://bugzilla.mozilla.org/show_bug.cgi?id=1525606#c13))